Fabric.Authorization.API/DependencyInjection/BaseSqlServerConfigurator.cs

```diff
@@ -20,6 +20,7 @@ public void ConfigureApplicationInstances(TinyIoCContainer container)
         {
             container.Register<IDbBootstrapper, SqlServerDbBootstrapper>().AsMultiInstance();
             RegisterDatabaseContext(container).AsMultiInstance();
+            RegisterEDWDatabaseContext(container).AsMultiInstance();
```

Comment:
Do we need an EdwDatabaseContext in the application container? We should only need this if we want to do work with the context outside the scope of a request, for instance at startup.

Reply:
Thats good to know, thanks.  We can remove it then.